implementations/c/test/ockam/transport/client/ipaddress.txt

```diff
@@ -0,0 +1,2 @@
+192.168.0.77
+8000
```

Comment:
Two things about this file:

1. since this file is committed to the code repo, keeping it separate from the test code doesn't seem very useful. It could instead be variables in the test code with default values that can be overridden with CLI arguments.

2. Was there a reason to chose `192.168.0.77` as a default value? I fear this test, if automated, will fail if the machine running the test doesn't have this subnet defined `192.168.*.*` ... could `127.0.0.1` or `0.0.0.0` be better defaults?

Reply:
If there are command line args I use those, otherwise I default to loopback address 127.0.0.1